app/assets/javascripts/class_lists/ClassListCreatorPage.story.js

```diff
@@ -4,13 +4,15 @@ import {storiesOf} from '@storybook/react';
 import ClassListCreatorPage from './ClassListCreatorPage';
 import mockWithFixtures from './fixtures/mockWithFixtures';
 import storybookFrame from './storybookFrame';
+import {testProps} from './ClassListCreatorPage.test';
 
 storiesOf('classlists/ClassListCreatorPage', module) // eslint-disable-line no-undef
```

Comment:
Curious to learn more about your development flow here! When you develop this feature, are you mostly looking at it in the context of the locally-hosted Rails app, or the Storybook, or a mix of both?

Reply:
Kind of a mix.  This might be a good thing to demo Friday or some other time!

I used a story as the primary way to develop the "create your lists" component, since I could set up the fixtures and then wanted to just iterate on the look and feel for a while.  I find stories super helpful for "view-only" kind of components.  For other components, I'll use the story to see them in different states at once.  Other times, when there's interactions with the server or it's complicated to do the fixture setup I would use local development like normal.

The other thing I tried with this work, which worked pretty well, was having the test files export their setup (eg, `testProps` here).  Then the story can re-use that.  And then it's also oneliner to add a snapshot test to ensure that changes don't regress how the Story is rendering.